tests/test_interestingness.py

```diff
@@ -238,12 +237,7 @@ def test_interestingness_0_2_0(global_var):
             and str(df.recommendation["Enhance"][f].mark) == "scatter"
         ):
             rank2 = f
-        if (
-            str(df.recommendation["Enhance"][f]._inferred_intent[2].attribute) == "Year"
-            and str(df.recommendation["Enhance"][f].mark) == "scatter"
-        ):
-            rank3 = f
-    assert rank1 < rank2 and rank1 < rank3 and rank2 < rank3
+    assert rank1 < rank2
```

Comment:
Is there a reason why the rank3 check was removed here? Was wondering if that specific chart was no longer being included in the Enhance action's recommendation list.

Reply:
I removed this because we are invalidating charts that have 2 measures and 1 temporal value (it doesn't make sense to "color" a temporal value), so this chart is no longer showing up in the Enhance recommendation list.

Reply:
this was reverted:)